src/noisepy/monitoring/monitoring_utils.py

```diff
@@ -50,6 +50,7 @@ class ConfigParameters_monitoring(BaseModel):
     ratio: float = Field(default=3.0, description="ratio for determining noise level by Mean absolute deviation (MAD)")
 
     # --- paramters for measuring attenuation
+    single_station: bool = Field(default=True, description="make measurement one one lag or two")
```

Comment:
```suggestion
    single_station: bool = Field(default=True, description="make measurement on one lag or two")
```

Reply:
Thank you for noticing this typo. I would need to update the description of this.